### DISCUSSION TITLE
Suggestion: 2D meshing for a slice image

### DISCUSSION THREAD

**Discussion by MEMPANE** [NREL/MATBOX_Microstructure_analysis_toolbox#25] [General] [answered]
Hello,

Sometimes, electrochemical simulation in 2-dimension is more than enough to demonstrate real-world phenomena.

Or, before getting into a 3-dimensional simulation, the computational cost of which is much greater than that of a 2-dimension, it is beneficial to first glance at the result of the 2-dimensional simulation, taking much less time.

Thus, it will be good if the MATBOX can support giving a 2D mesh for a slice image in a stack image.

Is it possible?

**Comment by fussegli** [comment, marked as answer]
Hello,
This would require to dig in Iso2mesh documentation. 2D mesh is much simpler than 3D so no doubt it is doable.
Although, I do not agree about the premise, especially for battery modeling. The transport path within the pore network of a microstructure is vastly different between the true 3D path and the restricted 2D path.
Because of this i have no plan in the near future to add 2D mesh. Eventually, if i have the the time, I will add it.
Best,
